src/completions.jl

```diff
@@ -1,73 +1,107 @@
 handle("completions") do data
-  @destruct [path || nothing,
-             mod || "Main",
-             editorContent || "",
-             lineNumber || 1,
-             startLine || 0,
-             column || 1,
-             line, force] = data
+  @destruct [
+    # general
+    line,
+    mod || "Main",
+    path || nothing,
+    # local context
+    context || "",
+    row || 1,
+    startRow || 0,
+    column || 1,
+    # configurations
+    force || false
+  ] = data
 
   withpath(path) do
```

Comment:
Not sure why do we need `withpath` here. Do you think we can remove this ?

Reply:
I *think* I wanted path completions to be relative to the file you're in, but `withpath` doesn't help with that afaict. So yeah, we can probably remove this.

Reply:
I get it. Umm maybe we need to modify `withpath` itself then ? If you want to leave this as is, I'm okay with that (relative path completions would be so great)